servicedirectory/endpoints_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicedirectory
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestEndpoint(t *testing.T) {
+	// <test setup code>
+	tc := testutil.SystemTest(t)
+	createNsBuf := new(bytes.Buffer)
+	createNsErr := createNamespace(createNsBuf, tc.ProjectID)
+	if createNsErr != nil {
```

Comment:
```suggestion
	if err := createNamespace(ioutil.Discard, tc.ProjectID); err != nil {
```

Doesn't look like the buffer is checked.